pandas/core/generic.py

```diff
@@ -8179,6 +8179,90 @@ def ranker(data):
 
         return ranker(data)
 
+    _shared_docs[
+        "differences"
+    ] = """
+        Compare to another %(klass)s and show the differences.
+
+        The axis on which to stack results and how much information to
+        preserve can be customized.
+
+        Note that NaNs are considered not different from other NaNs.
+
+        Parameters
+        ----------
+        other : %(klass)s
+            Object to compare with.
+
+        axis : {0 or 'index', 1 or 'columns'}, default 1
+            Determine how the differences are stacked.
```

Comment:
I'm hopeful the examples will clear this up, but I'm not able to predict what the ouput will look like based just on this description.

Reply:
I updated the description a bit and hopefully this would clear things up